### DISCUSSION TITLE
Inconvenience in use of jsonv5

### DISCUSSION THREAD

**Discussion by dyhkwong** [v2fly/v2ray-core#1603] [General]
- For `OutboundConfigurationObject`, only fields in `SimplifiedConfig` can be configured, features like Socks5 auth and VMess experiments are absent.
- `quic`, `http` (and `domainsocket`?) aren't registered as `transport`, so we cannot use these transports in `jsonv5`.

Any reasons about that? Are these consider as bugs or just because `jsonv5` is still WIP?

**Comment by AkinoKaede** [comment]
Yes, it is WIP currently.
